src/main/java/com/github/manolo8/darkbot/extensions/plugins/PluginHandler.java

```diff
@@ -260,5 +266,22 @@ private void testSignature(Plugin plugin, JarFile jar) throws IOException {
         }
     }
 
+    private String[] fetchFeatureList(Plugin plugin, JarFile jar) {
+        URLClassLoader loader = new URLClassLoader(new URL[]{ plugin.getJar() });
+        List<String> features = new ArrayList<>();
+        Enumeration<JarEntry> entries = jar.entries();
+        while (entries.hasMoreElements())
```

Comment:
```suggestion
        while (entries.hasMoreElements()) {
```
This is a code block, put some braces around it for readability